### PR TITLE
Add Flamelink 🔥🔗

### PR DESCRIPTION
**Pull request by jperasmus**

### CODE REVIEW

File: content/projects/flamelink.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+<img src="/images/flamelink-navigation.png" title="Flamelink Navigation">
+
+## Powered by Google's Firebase
+
+It's fast, secure, scalable and reliable. Flamelink leverages the power and features of Firebase to create a Headless, API Driven CMS to effortlessly manage your content. It's easy to use, so it's perfect for content managers and coders alike.
+
+## Customized Content
+
+<img src="/images/flamelink-bro.png" title="Flamelink: Do you even add and manage content, bro?">
```
Comment (erquhart) [inline content/projects/flamelink.md]: This image takes a lot of space, but distracts from the actual description of the CMS, should be removed.

File: content/projects/flamelink.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+Flamelink removes the guesswork by allowing you to create the content models you want via the CMS you need.
+
+## Flexible Frontends
+
+<img src="/images/flamelink-flexible.png" title="Flamelink Flexibility">
```
Comment (erquhart) [inline content/projects/flamelink.md]: Same as above, should be removed.

File: content/projects/flamelink.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+---
+title: Flamelink 🔥🔗
```
[the remark this answers is not in the document]
Comment (jperasmus) [reply content/projects/flamelink.md]: 👍

File: content/projects/flamelink.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+homepage: https://flamelink.io
+twitter: FlamelinkCMS
+opensource: "No"
+typeofcms: "Realtime Headless"
```
[the remark this answers is not in the document]
Comment (jperasmus) [reply content/projects/flamelink.md]: Updated, thanks

**Comment by jperasmus** [comment]
Thanks @erquhart, I've made the requested changes now

**Pull request merged**